modules/custom/az_core/az_core.module

```diff
@@ -266,7 +266,6 @@ function az_core_canonical_url(array $data) {
   $node = $data['node'];
   /** @var \Drupal\Core\Url $url */
   $url = $node->toUrl();
-  $url->setAbsolute(TRUE);
```

Comment:
This was originally `$uri = $url->toUriString();` before my PR. Should that be put back to original?

Reply:
👇 line 269